### PR TITLE
GLS edit of Priority Sliders completed

### PR DESCRIPTION
**Pull request by nmuller66**

### CODE REVIEW

File: docs/slides/content/priority-sliders.md [hunk 1]
```diff
@@ -112,10 +112,12 @@ How did you reach consensus?
...
 
 #### Facilitation Tips
-* Topics are not product features but product area are team-building topics
-* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritisation
-* The Customer team is the one defining priorities
-* it's not static - when the work we're doing takes us towards one priority, it may be that things can change. As such, treat this as a dynamic artefact
+* Topics are not product features but product area are team-building topics.
+<!--Tech Editor: I don't understand this first bullet, and "product area are team building topics" has a subject-verb agreement error and is unclear. The first part is ok, "topics are not product features", but the second part confused me. Please edit for better clarity. -->
+* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritization.
+<!--TechEdit: Does this mean "Bring priority slider artifacts to sprint planning to use when prioritizing items in the backlog." Please update. -->
```
Comment (ckavili) [inline docs/slides/content/priority-sliders.md:118]: exactly like this 🙈 happy to use your comment!

File: docs/slides/content/priority-sliders.md [hunk 2]
```diff
@@ -112,10 +112,12 @@ How did you reach consensus?
 
 
 #### Facilitation Tips
-* Topics are not product features but product area are team-building topics
-* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritisation
-* The Customer team is the one defining priorities
-* it's not static - when the work we're doing takes us towards one priority, it may be that things can change. As such, treat this as a dynamic artefact
+* Topics are not product features but product area are team-building topics.
+<!--Tech Editor: I don't understand this first bullet, and "product area are team building topics" has a subject-verb agreement error and is unclear. The first part is ok, "topics are not product features", but the second part confused me. Please edit for better clarity. -->
```
Comment (ckavili) [inline docs/slides/content/priority-sliders.md:116]: just asked for clarification. I'll update here later on :)

File: docs/slides/content/priority-sliders.md (hunk 1 above)
Comment (nmuller66) [inline docs/slides/content/priority-sliders.md:118]:
```suggestion

```

File: docs/slides/content/priority-sliders.md [hunk 3]
```diff
@@ -112,10 +112,12 @@ How did you reach consensus?
...
+* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritization.
```
Comment (nmuller66) [inline docs/slides/content/priority-sliders.md:117]:
```suggestion
* Bring priority slider artifacts to sprint planning to use when prioritizing items in the backlog.
```

File: docs/slides/content/priority-sliders.md (hunk 2 above)
Comment (springdo) [reply docs/slides/content/priority-sliders.md:116]: A quick scan but I think it was meant to say `Topics are not product features but product area or team-building topics`

File: docs/slides/content/priority-sliders.md [hunk 4]
```diff
@@ -70,52 +70,53 @@ _A practice that increases the autonomy of teams_
...
 ![what-is-it](images/priority-sliders/consensus-tools.png)<!-- .element: class="image-no-shadow image-full-width" -->
 
 
 
 #### Facilitation Tips
-* Topics are not product features but product area are team-building topics
-* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritisation
-* The Customer team is the one defining priorities
-* it's not static - when the work we're doing takes us towards one priority, it may be that things can change. As such, treat this as a dynamic artefact
+* Topics are not product features but product area are team-building topics.
```
Comment (ckavili) [inline docs/slides/content/priority-sliders.md:115]:
```suggestion
* Topics are not product features but product area or team-building topics.
```

File: docs/slides/content/priority-sliders.md (hunk 2 above)
Comment (ckavili) [reply docs/slides/content/priority-sliders.md:116]:
```suggestion
```

Thanks @springdo :) @nmuller66 let's update that sentence as Donal shared and remove the comment - then we can merge it :)

**Comment by nmuller66** [comment]
Great, committed Donal's change and the comment removal. This should be good to go!

**Pull request merged**